### DISCUSSION TITLE
报错

### DISCUSSION THREAD

**Discussion by asdghvh** [HuYo-OS/Genshin_auto_fish#27] [Q&A]
Traceback (most recent call last): 
  File "E:\2\ai\2\Genshin_auto_fish\fishing.py", line 9, in <module> 
    import torch 
ModuleNotFoundError: No module named

**Comment by LittleMio** [comment]
请确认你已经安装了Pytorch：
### 二选一
> 如果你是NVIDIA显卡并且已经正确安装了CUDA和cuDNN，请在环境中运行:
```shell
pip3 install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu117
```
如果你的显卡不是NVIDIA显卡 或 只想用CPU跑的，请在环境中运行:
```shell
pip3 install torch torchvision torchaudio
```
